invenio_cli/commands/services.py

```diff
@@ -255,6 +255,12 @@ def fixtures(self):
 
         return steps
 
+    def declare_queues(self):
```

Comment:
major: this will work for a local setup, it also needs to be included in the containerized version since they are `FunctionStep`.